source/index.txt

```diff
@@ -1,10 +1,141 @@
+   - Design an effective MongoDB schema, derived from an existing relational schema.
+   - Migrate data from Oracle, SQL Server, MySQL, PostgreSQL to MongoDB, 
+     while transforming to the target schema.
+   - Generate code artifacts to reduce the time required to update application code.
+
+   |
+
+   .. button:: Download Relational Migrator
+      :uri: http://mongodb.com/try/download/relational-migrator
+
+.. image:: /img/hero.png
+   :alt: Homepage hero image
+
+.. kicker:: What You Can Do
+
+Bring Your Relational Workloads to MongoDB With Confidence
+----------------------------------------------------------
+
+.. tabs::
+
+   .. tab:: Mapping and Modelling
+      :tabid: mapping-mod
+
+      .. procedure::
+
+         .. step:: Visualize Source and Destination Schemas
+
+            Use the entity-relationship style diagramming interface 
+            to visual your schema and make schema mapping decisions.
+
+         .. step:: Customize While Migrating
+
+            Split or merge tables across collections. 
+            Embed a table as a nested document or array in an existing collection.
+            Define custom JavaScript expressions to create additional fields.
+
+      .. image:: /img/tab1img.png
+         :alt: Mapping and Modelling
+
+   .. tab:: Schema Recommendations
+      :tabid: schema-rec
+
+      .. procedure::
+
+         .. step:: Get Suggested Schema Recommendations
+
+            Accelerate your schema mapping by asking Relational 
+            Migrator to suggest a MongoDB schema for your workload, 
+            or start with a blank slate.
+
+         .. step:: Apply your Mapping Rules
+
+            Quickly apply recommended table mappings 
+            into a specific collection in your model.
+
+      .. image:: /img/tab2img.png
+         :alt: Schema Recommendations
+
+   .. tab:: Data Synchronization
+      :tabid: data-sync
+
+      .. procedure::
+
+         .. step:: Replicate From Supported Relation Systems
+
+            Replicate data from Oracle, MySQL, SQL Server, or PostgreSQL 
+            to MongoDB (Atlas or on-premises).
+
+         .. step:: Transform Data as Its Replicated
+
+            Use the saved mapping definitions to transform data to 
+            the MongoDB schema as its replicated.
+            Replication can be one time or continuous.
+
+      .. image:: /img/tab3img.png
+         :alt: Data synchronization
+
+   .. tab:: Convert your Application Code
+      :tabid: convert-application-code
+
+      .. procedure::
+         
+         .. step:: Pick a Language and Template
+
+            Support for popular enterprise languages 
+            including C#, Java, JavaScript, JSON and multiple 
+            template frameworks.
+
+         .. step:: Automatically Generate Application Code
+
+            Use information known about the new schema to accelerate 
+            development of new code. Includes code generation of entity 
+            classes, persistence layers and APIs
```

Comment:
(punctuation x2)
s/n "... classes, persistence layers[,] and APIs[.]"

Reply:
Updated x2